gluon/dal.py

```diff
@@ -117,3 +117,11 @@ def _default_validators(db, field):
         DRIVERS['pg8000'] = pg8000
     except:
         pass
+if not DRIVERS.get('sqlite3'):
```

Comment:
Wait, was this supposed to stop that sqlite error when you launch web2py initially?